Makefile

```diff
@@ -0,0 +1,18 @@
+all: noembed
+
+submodule:
+	git submodule update --init
+
+noembed: submodule build
+
+generate:
+	go get github.com/jteeuwen/go-bindata
```

Comment:
This doesn't seem to build go-bindata. It seems to require a manual build with "make".